src/ussd_popi_faq.js

```diff
@@ -0,0 +1,329 @@
+go.app = function() {
+    var vumigo = require("vumigo_v02");
+    var SeedJsboxUtils = require('seed-jsbox-utils');
+    var Q = require('q');
+    var _ = require('lodash');
+    var MetricsHelper = require('go-jsbox-metrics-helper');
+    var App = vumigo.App;
+    var EndState = vumigo.states.EndState;
+    var ChoiceState = vumigo.states.ChoiceState;
+    var PaginatedChoiceState = vumigo.states.PaginatedChoiceState;
+    var PaginatedState = vumigo.states.PaginatedState;
+    var Choice = vumigo.states.Choice;
+    var JsonApi = vumigo.http.api.JsonApi;
+
+    var GoNDOH = App.extend(function(self) {
+        App.call(self, "state_start");
+        var $ = self.$;
+        var interrupt = true;
+        var utils = SeedJsboxUtils.utils;
+
+        var is;
+        var sbm;
+        var hub;
+        var ms;
+
+        self.init = function() {
+            // initialise services
+            is = new SeedJsboxUtils.IdentityStore(
+                new JsonApi(self.im, {}),
+                self.im.config.services.identity_store.token,
+                self.im.config.services.identity_store.url
+            );
+            sbm = new SeedJsboxUtils.StageBasedMessaging(
+                new JsonApi(self.im, {}),
+                self.im.config.services.stage_based_messaging.token,
+                self.im.config.services.stage_based_messaging.url
+            );
+            hub = new SeedJsboxUtils.Hub(
```

Comment:
It doesn't look like you use the Hub at all (this is where we store registrations) so you don't need to set up this connection to it